reporter/src/main/kotlin/reporters/NoticeTemplateReporter.kt

```diff
@@ -166,7 +167,20 @@ class NoticeTemplateReporter : Reporter {
     /**
      * A collection of helper functions for the Freemarker templates.
      */
-    class NoticeHelper(private val licenseConfiguration: LicenseConfiguration) {
+    class NoticeHelper(private val ortResult: OrtResult, private val licenseConfiguration: LicenseConfiguration) {
+        /**
+         * Filter [packages] that are a dependency of at least one of the provided [projects][projectIds].
```

Comment:
Turning the preceeding "Filter" into "Return" would IMO be more clear about the filter criteria. 
As-is it is easier to understand it as the inversion of the criteria IMO.

Reply:
Done.